packages/shared/src/types/firestore.ts

```diff
@@ -25,7 +25,12 @@ export interface OrganizationData {
    * If not provided, will fall back to organization name
    * @TODO this isn't yet functional, apply when making email per organization
    */
-  emailFrom?: string;
+  emailFrom: string;
+  /**
+   * Email to send a copy of sent Email.
+   * If not provided, will fall back to emailFrom
+   */
+  bcc: string;
```

Comment:
1. `emailFrom` might or might not be provided to the organisation info, same as `bcc`. 
2. Can we rename the `bcc` to `emailBcc` (as this is the general organisation info so there's not much context for the `bcc` field and what it represents